source/session.ts

```diff
@@ -545,7 +545,7 @@ export class Session<
     const mergedEntity = identityMap[identifier];
 
     for (const key in entity) {
-      if (entity.hasOwnProperty(key)) {
+      if (Object.prototype.hasOwnProperty.call(entity, key)) {
```

Comment:
eslint - avoid calling hasOwnProperty on the object directly

Reply:
Use [Object.hasOwn()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn) instead.

Reply:
fixed!